linux/ibus-keyman/src/engine.c

```diff
@@ -958,7 +956,7 @@ ibus_keyman_engine_process_key_event(
   km_kbp_context *context = km_kbp_state_context(keyman->state);
   g_free(get_current_context_text(context));
   g_message("DAR: %s - km_mod_state=0x%x", __FUNCTION__, km_mod_state);
-  km_kbp_status event_status = km_kbp_process_event(keyman->state, keycode_to_vk[keycode], km_mod_state, isKeyDown);
+  km_kbp_process_event(keyman->state, keycode_to_vk[keycode], km_mod_state, isKeyDown);
```

Comment:
We should be testing the return value from `km_kbp_process_event` and logging failures -- so I'm not that excited about this fix...